transformer_engine/pytorch/module/base.py

```diff
@@ -98,42 +98,38 @@ def initialize_ub(
     assert _ub_communicators is None, "UB communicators are already initialized."
     _ub_communicators = {}
 
```

Comment:
The PR fixes a bug with DGRAD-RS overlap. https://github.com/NVIDIA/TransformerEngine/pull/1088 
Please make sure that PR changes are not reverted with this change.